youtube_dlc/downloader/hls.py

```diff
@@ -211,6 +222,14 @@ def is_ad_fragment_end(s):
                 elif is_ad_fragment_end(line):
                     ad_frag_next = False
 
+        if external_downloader == 'aria2c':
+            ctx['url_list'] = fragment_urls
+            ctx['decrypt_info'] = decrypt_info
+            self.urlopen = self.ydl.urlopen
```

Comment:
Why? I don't see `self.urlopen` being used anywhere

Reply:
There was an error earlier regarding that, but it's been fixed now, forgot to remove it.